lib/spack/spack/test/spec_semantics.py

```diff
@@ -1129,18 +1129,22 @@ def _check_spec_format_path(spec_str, format_str, expected, path_ctor=None):
     "spec_str,format_str,expected",
     [
         (
-            "zlib@git.foo/bar",
+            "git-test@git.foo/bar",
             r"C:\\installroot\{name}\{version}",
-            r"C:\installroot\zlib\git.foo_bar",
+            r"C:\installroot\git-test\git.foo_bar",
         ),
         (
-            "zlib@git.foo/bar",
+            "git-test@git.foo/bar",
             r"\\hostname\sharename\{name}\{version}",
-            r"\\hostname\sharename\zlib\git.foo_bar",
+            r"\\hostname\sharename\git-test\git.foo_bar",
         ),
         # Windows doesn't attribute any significance to a leading
         # "/" so it is discarded
-        ("zlib@git.foo/bar", r"/installroot/{name}/{version}", r"installroot\zlib\git.foo_bar"),
+        (
+            "git-test@git.foo/bar",
+            r"/installroot/{name}/{version}",
+            r"installroot\git-test\git.foo_bar",
+        ),
     ],
 )
 def test_spec_format_path_windows(spec_str, format_str, expected):
```

Comment:
Does this also need `mock_git_test_package`?

Reply:
yeah it should -- thanks!

Reply:
fixed and pushed.